lib/Raven/Client.php

```diff
@@ -132,7 +152,11 @@ public function __construct(Configuration $config, TransportInterface $transport
         $this->config = $config;
         $this->transport = $transport;
         $this->processorRegistry = new ProcessorRegistry();
-        $this->context = new Context();
+        $this->userContext = new Context();
+        $this->tagsContext = new Context();
+        $this->extraContext = new Context();
+        $this->runtimeContext = new Context();
+        $this->serverOsContext = new Context();
```

Comment:
Why `serverOs` and not simply `server`? Is it that specific?

Reply:
I saw it in the Raven Ruby SDK and copied it here

https://github.com/getsentry/raven-ruby/blob/master/lib/raven/context.rb#L16